docker-compose.yml

```diff
@@ -59,7 +59,9 @@ services:
     createbuckets:
         image: minio/mc
         depends_on:
-            - minio
+            minio:
```

Comment:
Is it correct that `depends_on` now is not an array (of objects) anymore? But actually I trust that you have verified this, I'm just surprised.

I'm also surprised that it's `createbuckets` instead of `create-buckets`, but that's another story.

Reply:
> Is it correct that `depends_on` now is not an array (of objects) anymore? But actually I trust that you have verified this, I'm just surprised.

Yep. The key of the object is the name of the service that is depended on, so it's like:

```
[service]:
  condition: service_healthy
```